README.md

```diff
@@ -62,7 +62,7 @@ Now that you have FedScale installed, you can start exploring FedScale following
 
 FedScale consists of 20+ large-scale, heterogeneous FL datasets covering computer vision (CV), natural language processing (NLP), and miscellaneous tasks. 
 Each one is associated with its training, validation, and testing datasets. 
-Please go to the `./benchmark/dataset` directory and follow the dataset [README](./benchmark/dataset/README.md) for more details.
+Please go to the `./benchmark/dataset` directory and follow the dataset [README](./benchmark/dataset/README.md) for more details, in which we also acknowledge the dataset.
```

Comment:
Change to "We acknowledge the contributors of these raw datasets. Please check each dataset for more details."?